agent/src/agentclient.rs

```diff
@@ -285,6 +325,7 @@ impl<T: APIServerClient> BrupopAgent<T> {
         if !shadow_exists {
             self.create_metadata_shadow().await?;
         }
+        self.register_node_to_lb().await?;
```

Comment:
We may want to consider that Brupop users could be modifying this label outside of Brupop itself. I'm not totally convinced we should remove the exclusion label at the start, perhaps only if we know we have already added the label. What do you think?

Reply:
This was my mistake, this is the part didn't get clean properly when I was testing the functionality initially. Yes, we need to remove this part out.